client/src/javascript/components/torrent-list/TorrentList.js

```diff
@@ -313,6 +326,15 @@ class TorrentListContainer extends React.Component {
     });
   }
 
+  handleTorrentDownload(torrent, event) {
+    event.preventDefault();
+    const baseURI = ConfigStore.getBaseURI();
+    let link = document.createElement('a');
+    link.download = `${torrent.name}.tar`;
```

Comment:
There's an `isMultiFile` property on the `torrent` object — would you mind checking for that here, and adjusting the `download` property accordingly?

E.g.
```javascript
link.download = torrent.isMultiFile ? `${torrent.name}.tar` : torrent.name;
```

Reply:
Fixed!